bundles/statistics/statsgrid2016/plugin/ClassificationToolPlugin.js

```diff
@@ -108,7 +110,9 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.plugin.ClassificationToolPlugin
                 return me._element;
             }
             me._element = me._templates.main.clone();
-            this.__legend.render(me._element);
+            this.__legend.render(me._element, function(){
+                me._calculatePluginSize();
+            });
```

Comment:
With the Legend as observable change this would be:

     this.__legend.on('change', function(){
           me._calculatePluginSize();
     });